themes/openy_themes/openy_carnation/templates/node/blog/node--blog--teaser.html.twig

```diff
@@ -0,0 +1,110 @@
+ *     name will often be in a short form of the human readable label.
+ *   - node--view-mode-[view_mode]: The View Mode of the node; for example, a
+ *     teaser would result in: "node--view-mode-teaser", and
+ *     full: "node--view-mode-full".
+ *   The following are controlled through the node publishing options.
+ *   - node--promoted: Appears on nodes promoted to the front page.
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @todo Remove the id attribute (or make it a class), because if that gets
+ *   rendered twice on a page this is invalid CSS for example: two lists
+ *   in different view modes.
+ */
+#}
+{% set classes = [
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  node.field_blog_style.value ? 'blog-' ~ node.field_blog_style.value|clean_class : 'blog-story',
+] %}
+{% set category_default = node.bundle|clean_class|capitalize %}
+{% set category = content.field_blog_category.0 ? content.field_blog_category.0 : category_default|t %}
+{% set color = 'color: #' ~ content.field_blog_text_color['#items'].entity.field_color.value %}
+{% set bg_color = 'background-color: #' ~ content.field_blog_color['#items'].entity.field_color.value %}
+
+<article{{ attributes.addClass(classes) }}>
+  <div class="inner-wrapper" style="{{ color }}; {{ bg_color }};">
+    {% if ( node.field_blog_style.value == "photo" ) %}
```

Comment:
instead of direct node access use content variable. Otherwise you'll have bad security and lack of field_ui management